lib/msal-browser/src/config/Configuration.ts

```diff
@@ -122,7 +123,8 @@ export function buildConfiguration({ auth: userInputAuth, cache: userInputCache,
         postLogoutRedirectUri: "",
         navigateToLoginRequestUrl: true,
         clientCapabilities: [],
-        protocolMode: ProtocolMode.AAD
+        protocolMode: ProtocolMode.AAD,
+        azureCloudInstance: 1
```

Comment:
Can we use the enum here rather than the index?

Reply:
Done